control/timeresp.py

```diff
@@ -210,40 +210,45 @@ def forced_response(sys, T=None, U=0., X0=0., transpose=False,
 
     T : array_like, optional for discrete LTI `sys`
         Time steps at which the input is defined; values must be evenly spaced.
+        If None, `U` must be given and and `len(U)` time steps of sys.dt are
+        simulated. If sys.dt is None or True (undetermined time step), a dt
+        of 1.0 is assumed.
 
     U : array_like or float, optional
-        Input array giving input at each time `T` (default = 0).
+        Input array giving input at each time `T`
```

Comment:
I think that numpydoc style requires that you define the default value for optional arguments (someplace).  Also, missing a period at the end.

Reply:
https://numpydoc.readthedocs.io/en/latest/format.html

> Optional keyword parameters have default values, which are displayed as part of the function signature. They **can** also be detailed in the description

(emphasis by me)